src/global/state/globalActions.js

```diff
@@ -0,0 +1,22 @@
+export const GLOBAL_ACTION_TYPES = {
+  SET_AUTH_TOKEN: 'SET_AUTH_TOKEN',
+  SET_USER_ID: 'SET_USER_ID'
+};
+
+export function setAuthToken(token) {
```

Comment:
You can combine this with the setUserId function since they're always going to be changed at the same time (login)